Extensions/TileMap/pixi-tilemap-helper.js

```diff
@@ -438,33 +555,41 @@
     tiledData,
     atlasImageResourceName,
     tilemapResourceName,
-    tilesetResourceName
+    tilesetResourceName,
+    levelIndex
   ) => {
     const requestedTileMapDataId =
       tilemapResourceName +
       '@' +
       tilesetResourceName +
       '@' +
-      atlasImageResourceName;
+      atlasImageResourceName +
+      '@' +
+      levelIndex;
 
     // If the tilemap data is already in the cache, use it directly.
+    // For LDtk we do not need to generate a tileset
+
+    // TODO we will need to split the cache here - one for tilemaps and another for tilesets
+    // TODO this is because LDtk tilemaps can have different atlas per layer
```

Comment:
As this TODO is not done yet, I guess there can be problems if we import a LDtk tilemap that has different atlas per layer? How is the problem showing?

Reply:
ah its just a comment I forgot to remove